tests/general/InvalidLinkBearTest.py

```diff
@@ -404,7 +411,7 @@ def test_links_to_ignore(self):
                            'http://www.notexample.com/404',
                            'http://exampe.com/404',
                            'http://example.co.in/404'
-                          ]
+                           ]
```

Comment:
unrelated change?  should be a separate commit, or a 'newcomer' issue if it isnt critical.